### PR TITLE
Add resize options

### PR DESCRIPTION
**Pull request by Mirobit**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -70,6 +70,17 @@ Type: `Buffer`
 
 Buffer to optimize.
 
+##### resize
+
+Type: `string`
+
+Resize the output GIF to the given width and height (`widthxheight`)
+
+##### resizeHeight / resizeWidth
```
Comment (sindresorhus) [inline readme.md]: These should be separate items in the docs.

**Comment by Extarys** [comment]
I support the inclusion of this option. Imagemagick animated gif resizing options are bad.

**Comment by sindresorhus** [comment]
No need for 3 options. Just add a resize option that is an object with a width and height property. If only one property is given, it should use the aspect ratio flag, otherwise the `—resize` flag.

**Comment by sindresorhus** [comment]
Closing for lack of response.

**Pull request closed without merging**